.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell flake8 flake8-2020
+                         flake8-bugbear flake8-comprehensions
+                         mypy pyupgrade safety
+      - run: bandit --recursive  --skip B101,B307,B404,B603 .
```

Comment:
Would it make sense to not skip any checks globally and instead [mark the failing lines with `# nosec`](https://bandit.readthedocs.io/en/latest/config.html#suppressing-individual-lines) when we know we've audited and accepted the check violation?

Reply:
I added `# nosec` to individual lines.